edgexfoundry/app-rfid-llrp-inventory
# refactor!: Replace internal topics from config with new constants and use base topic

BREAKING CHANGE: Internal topics no longer configurable, except the base topic. Trigger topics for edgex-messagebus and external-mqtt now directly under Trigger section. All configured topics (subscribe and function pipeline) now automatically have the base topic (default of "edgex/") prepended.







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-rfid-llrp-inventory/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  TBD

## Testing Instructions
Run non-secure edgex stack with Device Virtual and MQTT Broker 
build  and run service from this branch with `-o -cp` options
Verify from logs has following messages:
```
"Waiting for messages from the MessageBus on the 'edgex/edgex/events/+/device-rfid-llrp/+/+/ReaderEventNotification' topic"
"Waiting for messages from the MessageBus on the 'edgex/events/+/device-rfid-llrp/+/+/ROAccessReport' topic
```

## New Dependency Instructions (If applicable)
